### PR TITLE
Last-minute cleanup

### PR DESCRIPTION
**Pull request by aceyoung9**
Cleaning up after my 2014 work. Mostly clarifying docs and making this mildly more visually accessible.

### CODE REVIEW

File: _harp/_data.json [hunk 1]
```diff
@@ -1,51 +1,49 @@
...
          "Java",
          "Python",
          "Swift",
          "LaTeX"
       ]
    },
-   {  
+   {
       "name":"Danielle Nguyen",
-      "twitter":"My tumblr is http://smurtleturtle.tumblr.com/ bc I'm cultured and refined",
```
Comment (aceyoung9) [inline _harp/_data.json:24]: @daniellenguyen this whole string was getting piped into an `<a>` tag. Feel free to fix, or add a Tumblr attribute to the template.

File: _harp/_data.json [hunk 2]
```diff
@@ -1,51 +1,49 @@
-[  
-   {  
+[
+   {
       "name":"Jack Frysinger",
       "twitter":"jackfrys",
-      "www":"jackfrysinger.com",
-      "desc":"",
+      "www":"//jackfrysinger.com",
```
Comment (aceyoung9) [inline _harp/_data.json:5]: @jackfrys also I've never gotten this URL to load; did u type this in right?

File: _harp/_data.json (hunk 1 above)
Comment (daniellenguyen) [reply _harp/_data.json:24]: yes i was being a lazy butt. will do

**Comment by daniellenguyen** [comment]
do u need me to pore over this or is a skim enough bb

**Comment by aceyoung9** [comment]
check it as much or as little as you want. But merge it before you do the gh-pages publishing.

**Comment by daniellenguyen** [comment]
okays

**Pull request merged**